requirements-dev.txt

```
@@ -15,4 +15,5 @@ clang-format==13.0.0
 mypy>=0.971
 types-protobuf==3.18.4
 black>=22.3
-isort[colors]>=5.10
+isort>=5.10.1
+colorama==0.4.5
```

Is it OK to use `>=` instead? 
```suggestion
colorama>=0.4.5
```

---

Yes, >= makes more sense here. @justinchuby I see you are setting up fixed version for lint in onnx-script. Is there any concerns that we use >= ?

---

Not really. whatever works